dali/ft/filecopy.cpp

```diff
@@ -397,10 +397,31 @@ bool FileTransferThread::launchFtSlaveCmd(const SocketEndpoint &ep)
     }
     else
     {
-        setDafsEndpointPort(connectEP);
-        if (connectEP.isNull())
-            return false;
-        socket.setown(connectDafs(connectEP, 5000));
+        if (!isContainerized())
+        {
+            setDafsEndpointPort(connectEP);
+            if (connectEP.isNull())
+                return false;
+        }
+        // allow a relatively long time in case service is overloaded (unable to cope with # parallel requests)
+        constexpr unsigned timeoutMs = 1000*60*30; // 30 mins
+        CTimeMon tm(timeoutMs);
+        unsigned remaining;
+        while (true)
```

Comment:
introduces a retry and log loop. The previous short 5 second connect attempt could timeout if K8s service was throttling connections.